<ui version="4.0" >
 <class>FullScreenConfigDialog</class>
 <widget class="QWidget" name="FullScreenConfigDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>424</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="mSlideShowTitle" >
     <property name="text" >
      <string>Slideshow</string>
     </property>
     <property name="isTitle" stdset="0" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Interval:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QSlider" name="mSlideShowIntervalSlider" >
       <property name="minimum" >
        <number>2</number>
       </property>
       <property name="maximum" >
        <number>60</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mSlideShowIntervalLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" >
    <widget class="QCheckBox" name="mSlideShowLoopCheckBox" >
     <property name="text" >
      <string>Loop</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QCheckBox" name="mSlideShowRandomCheckBox" >
     <property name="text" >
      <string>Random</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QLabel" name="mMetaDataTitle" >
     <property name="text" >
      <string>Metadata</string>
     </property>
     <property name="isTitle" stdset="0" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QPushButton" name="mConfigureDisplayedInformationButton" >
       <property name="text" >
        <string>Configure Displayed Metadata...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Appearance</string>
     </property>
     <property name="isTitle" stdset="0" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Theme:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1" >
    <widget class="QListWidget" name="mThemeListWidget" />
   </item>
   <item row="8" column="1" >
    <widget class="QCheckBox" name="mShowThumbnailsCheckBox" >
     <property name="text" >
      <string>Show thumbnails</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout_4" >
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Height:</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="mHeightSlider" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>204</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum" >
        <number>70</number>
       </property>
       <property name="maximum" >
        <number>256</number>
       </property>
       <property name="value" >
        <number>70</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="2" >
    <widget class="Line" name="line" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>3</height>
      </size>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mCloseButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mSlideShowIntervalSlider</tabstop>
  <tabstop>mSlideShowLoopCheckBox</tabstop>
  <tabstop>mSlideShowRandomCheckBox</tabstop>
  <tabstop>mConfigureDisplayedInformationButton</tabstop>
  <tabstop>mThemeListWidget</tabstop>
  <tabstop>mShowThumbnailsCheckBox</tabstop>
  <tabstop>mHeightSlider</tabstop>
  <tabstop>mCloseButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mCloseButton</sender>
   <signal>clicked()</signal>
   <receiver>FullScreenConfigDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>325</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel" >
     <x>335</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mShowThumbnailsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>mHeightSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>137</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel" >
     <x>156</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mShowThumbnailsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_5</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>94</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel" >
     <x>85</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
